src/raft.rs

```diff
@@ -2107,19 +2096,21 @@ impl<T: Storage> Raft<T> {
     ///
     /// ```rust
     /// use raft::{Raft, Config, storage::MemStorage, eraftpb::ConfState};
-    /// let config = Config {
+    /// use raft::raw_node::new_mem_raw_node;
+    /// let mut config = Config {
     ///     id: 1,
     ///     peers: vec![1],
     ///     ..Default::default()
     /// };
-    /// let mut raft = Raft::new(&config, MemStorage::default()).unwrap();
+    /// let mut node = new_mem_raw_node(&mut config, MemStorage::default()).unwrap();
```

Comment:
We shouldn't show users how to use our library with testing only functions.

Please revert this.